File: JAVA/Pertemuan 01/src/id/ac/uph/pbo/pertemuan01/PersegiPanjang.java
package id.ac.uph.pbo.pertemuan01;

public class PersegiPanjang {
    private double panjang, lebar;
    public PersegiPanjang(double panjang, double lebar) {
        this.panjang = panjang; this.lebar = lebar;
    }
    public double hitungLuas()     { return panjang * lebar; }
    public double hitungKeliling() { return 2 * (panjang + lebar); }
    public void cetak() {
        System.out.printf("  PersegiPanjang (%.1fx%.1f): Luas=%.2f, Keliling=%.2f%n",
                panjang, lebar, hitungLuas(), hitungKeliling());
    }
}

File: PYTHON/Pertemuan 01/PersegiPanjang.py
import sys
sys.stdout.reconfigure(encoding='utf-8')
class PersegiPanjang:
    def __init__(self, panjang: float, lebar: float):
        self.panjang = panjang
        self.lebar = lebar

    def hitung_luas(self) -> float:
        return self.panjang * self.lebar

    def hitung_keliling(self) -> float:
        return 2 * (self.panjang + self.lebar)

    def cetak(self):
        print(f"  PersegiPanjang ({self.panjang:.1f}x{self.lebar:.1f}): Luas={self.hitung_luas():.2f}, Keliling={self.hitung_keliling():.2f}")

